blueprints/data-solutions/data-playground/main.tf

```diff
@@ -74,7 +74,8 @@ module "project" {
     "servicenetworking.googleapis.com",
     "stackdriver.googleapis.com",
     "storage.googleapis.com",
-    "storage-component.googleapis.com"
+    "storage-component.googleapis.com",
+    "dataproc.googleapis.com"
```

Comment:
Any specific reason to activate `dataproc` service? Is it mandatory to run this blueprint?